WooCommerce/Classes/ViewRelated/Products/Edit Product/Product Settings/ProductSettingsRows.swift

```diff
@@ -206,7 +206,20 @@ enum ProductSettingsRows {
              cell.isOn = settings.downloadable
              cell.onChange = { newValue in
                  //TODO: Add analytics M5
-                 self.settings.downloadable = newValue
+                 if !newValue {
+                     cell.isOn = true
```

Comment:
> I thought about it, but IMO it's just a nice to have, and it would require significant work (we need to expose the product to the settings Model), so I left it out, let me know what you think.

Please feel free to file a separate issue for this if it takes significant effort. Thanks!

Reply:
Done https://github.com/woocommerce/woocommerce-ios/issues/11406